schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-2

```diff
@@ -0,0 +1,90 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for mobile contexts",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "mobile_context",
+		"format": "jsonschema",
+		"version": "1-0-2"
+	},
+	"type": "object",
+	"properties": {
+		"osType": {
+			"type": "string"
+		},
+		"osVersion": {
+			"type": "string"
+		},
+		"deviceManufacturer": {
+			"type": "string"
+		},
+		"deviceModel": {
+			"type": "string"
+		},
+		"carrier": {
+			"type": ["string", "null"]
+		},
+		"networkType": {
+			"type": ["string", "null"],
+			"enum": ["mobile", "wifi", "offline"]
+		},
+		"networkTechnology": {
+			"type": ["string", "null"]
```

Comment:
This field (and some others) was `"type": "string"`. I wonder if the added "null" option can be a breaking change.
Happy to have a feedback on this by the experts.

Reply:
This isn't a breaking change, you are widening the allowed values so that's fine. Even if this was in the `required` array at the bottom in 1-0-1 then I think that'd be fine as you'd be going from a NOT NULL to NULL (although I'm happy to be corrected on this, but I think we're good here).